### PR TITLE
DDCE-789:DeadEnd Page creation

### PR DESCRIPTION
**Pull request by manpreetsingh1000**
# DDCE-(789)

##Bug fix / New feature (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

**Comment by abhijeetardale** [comment]
Linked PR
https://github.com/hmrc/charities-registration-journey-tests/pull/52/files

File: app/views/DeadEndView.scala.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    @h1(messages("deadEnd.heading"))
+
+    @p{@messages("deadEnd.p1")}
+    @p{@messages("deadEnd.p2")}
+
+    @link(controllers.routes.IndexController.onPageLoad().url, "site.gotoTaskList")
+
+
+}
```
Comment (robert-webb-hmrc) [inline app/views/DeadEndView.scala.html:43]: New line required here

File: app/views/components/linkAsButton.scala.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+@(
+    link: String,
+    messageKey: String,
+    id: Option[String] = None,
+)(implicit messages: Messages)
+
+<a href="@link" role="button" class="govuk-button govuk-button" @id.map(x => s"id=$x")>
+    @messages(messageKey)
+</a>
```
Comment (robert-webb-hmrc) [inline app/views/components/linkAsButton.scala.html:27]: New line required here

File: app/views/DeadEndView.scala.html (hunk 1 above)
Comment (manpreetsingh1000) [reply app/views/DeadEndView.scala.html]: Done and pushed it up

File: app/views/components/linkAsButton.scala.html (hunk 2 above)
Comment (manpreetsingh1000) [reply app/views/components/linkAsButton.scala.html]: Done and pushed it up

**Pull request merged**